src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php

```diff
@@ -23,8 +23,9 @@ class Shopgate_Framework_Model_Payment_Braintr_Abstract
     extends Shopgate_Framework_Model_Payment_Abstract
     implements Shopgate_Framework_Model_Payment_Interface
 {
-    const ADVANCED_FRAUD_REVIEW  = 'Review';
-    const ADVANCED_FRAUD_DECLINE = 'Decline';
+	const ADVANCED_FRAUD_REVIEW              = 'Review';
```

Comment:
I don't believe we have `=` alignments in framework, the tab characters are also used here?

Reply:
Actually we are aligning them in the code (most of the time), but not always for constants. I would recommend that we should do it all the times. Objections?

Reply:
no objections here